### PR TITLE
Commissions UI fixes

### PR DESCRIPTION
**Pull request by chaotixkilla**

### CODE REVIEW

**Comment by chaotixkilla** [comment]
[Commission fixes](https://trello.com/c/zTreTcii/579-commission-fixes)

File: new-lamassu-admin/src/pages/Commissions/helper.js [hunk 1]
```diff
@@ -63,6 +68,12 @@ const onCryptoChange = (prev, curr, setValue) => {
   setValue(curr)
 }
 
+const boldStyle = () => {
+  return {
+    fontWeight: 'bold'
+  }
+}
+
```
Comment (josepfo) [inline new-lamassu-admin/src/pages/Commissions/helper.js:74]: What do you think about using the monoBold in the typography\styles.js here?

**Review by josepfo — CHANGES_REQUESTED** [review]

**Review by josepfo — APPROVED** [review]

File: new-lamassu-admin/src/pages/Commissions/helper.js [hunk 2]
```diff
@@ -39,7 +39,12 @@ const cashOutHeader = (
 const getView = (data, code, compare) => it => {
   if (!data) return ''
 
-  return R.compose(R.prop(code), R.find(R.propEq(compare ?? 'code', it)))(data)
+  // The following boolean should come undefined if it is rendering an unpaired machine
+  const hasValue = R.find(R.propEq(compare ?? 'code', it))(data)
+
+  return hasValue
+    ? R.compose(R.prop(code), R.find(R.propEq(compare ?? 'code', it)))(data)
```
Comment (RafaelTaranto) [inline new-lamassu-admin/src/pages/Commissions/helper.js]: Why not just do R.prop(code) of hasValue?

**Review by RafaelTaranto — CHANGES_REQUESTED** [review]

File: new-lamassu-admin/src/pages/Commissions/helper.js (hunk 2 above)
Comment (chaotixkilla) [reply new-lamassu-admin/src/pages/Commissions/helper.js]: You're correct, fixing

File: new-lamassu-admin/src/components/layout/TitleSection.styles.js [hunk 3]
```diff
@@ -15,6 +17,10 @@ export default {
   subpageButton: {
     marginLeft: 12
   },
+  buttonText: {
+    color: `${backgroundColor}`,
```
Comment (RafaelTaranto) [inline new-lamassu-admin/src/components/layout/TitleSection.styles.js]:
```suggestion
    color: backgroundColor,
```

**Review by RafaelTaranto — CHANGES_REQUESTED** [review]

**Pull request merged**